sample/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,8 @@
     <string name="theme_color_test_for_xml">theme color test for xml</string>
     <string name="extended_color_test_for_compose">extended color test for compose</string>
     <string name="extended_color_test_for_xml">extended color test for xml</string>
+
+    <string name="request_permission_dialog_content">Hi, Please allow APP to access your location. When location permission is enabled, you can see what\'s new around you.</string>
+    <string name="request_permission_dialog_cancel_button">Not Now</string>
+    <string name="request_permission_dialog_confirm_button">OK</string>
```

Comment:
Could we move these two action label(OK and Not Now) to top of the strings.xml
Rename them to `action_label_ok` & `action_label_not_now`